build.gradle.kts

```diff
@@ -28,3 +30,10 @@ tasks {
         }
     }
 }
+
+tasks.dokkaHtmlMultiModule.configure {
+    pluginsMapConfiguration.put(
+        "org.jetbrains.dokka.base.DokkaBase",
+        """ { "footerMessage": "Copyright ${LocalDate.now().year} Exactpro Systems Limited" } """,
```

Comment:
```suggestion
        """ { "footerMessage": "Copyright 2022-${LocalDate.now().year} Exactpro Systems Limited" } """,
```

I'd use utc date

Reply:
Add an Redmine task to move this code to the conventions plugin

Reply:
> Add an Redmine task to move this code to the conventions plugin

Or TODO in code

Reply:
Created issue RM#82866